Makefile

```diff
@@ -174,4 +175,10 @@ api-user:
 
 .PHONY: api-admin
 api-admin:
-	go run cmd/temporal/main.go $(TEMPORALDEVFLAGS) admin $(USER)
\ No newline at end of file
+	go run cmd/temporal/main.go $(TEMPORALDEVFLAGS) admin $(USER)
+
+.PHONY: setup
```

Comment:
Should this auto execute when `make` is called (so part of the `all` routine?) 

(not super familiar with make flow TBH)

Reply:
yea it should I've added that change